kge/job/job.py

```diff
@@ -55,36 +58,92 @@ def __init__(self, config: Config, dataset: Dataset, parent_job: "Job" = None):
             for f in Job.job_created_hooks:
                 f(self)
 
-    def resume(self, checkpoint_file: str = None):
-        """Load job state from last or specified checkpoint.
-
-        Restores all relevant state to resume a previous job. To run the restored job,
-        use :func:`run`.
+    @staticmethod
+    def create(
+        config: Config, dataset: Optional[Dataset] = None, parent_job=None, model=None
+    ):
+        from kge.job import TrainingJob, EvaluationJob, SearchJob
 
-        Should set `resumed_from_job` to the job ID of the previous job.
+        if dataset is None:
+            dataset = Dataset.load(dataset)
```

Comment:
This looks shaky. What are we loading if dataset is None?

Reply:
ah, supposed to be config